crawler/src/main/java/com/movie/recommender/crawler/service/MovieService.java

```diff
@@ -1,18 +1,89 @@
 package com.movie.recommender.crawler.service;
 
 import com.movie.recommender.common.client.LocationServiceClient;
+
+import com.movie.recommender.common.model.movie.MovieCredits;
+import com.movie.recommender.common.model.movie.MovieDetails;
+import com.movie.recommender.common.model.movie.MovieKeywords;
+import com.movie.recommender.common.model.movie.NowPlayingMoviesByCountry;
+import com.movie.recommender.crawler.client.SerpApiClient;
+import com.movie.recommender.crawler.client.TmdbApiClient;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.stereotype.Service;
 
+import java.util.*;
+
 @Service
 @Slf4j
 public class MovieService {
     private final LocationServiceClient locationServiceClient;
+    private final TmdbApiClient tmdbApiClient;
+    private final SerpApiClient serpApiClient;
+    private final MongoDBService mongoDBService;
+    private final String DEFAULT_COUNTRY = "ua";
 
-    public MovieService(LocationServiceClient locationServiceClient) {
+    public MovieService(LocationServiceClient locationServiceClient, MongoDBService mongoDBService) {
         this.locationServiceClient = locationServiceClient;
+        this.tmdbApiClient = new TmdbApiClient();
+        this.serpApiClient = new SerpApiClient();
+        this.mongoDBService = mongoDBService;
+    }
+
+    public void loadNowPlayingMovies() {
+        String country = DEFAULT_COUNTRY;
+        Optional<NowPlayingMoviesByCountry> nowPlayingMovie = tmdbApiClient.getNowPlayingMovies(country);
+        if (nowPlayingMovie.isPresent()) {
+            nowPlayingMovie.get().setCountry(country);
+            mongoDBService.insertNowPlayingMovies(nowPlayingMovie.get());
+        } else {
+            log.warn("No now playing movies found for country '{}'.", country);
+        }
+
+
     }
 
+
+    public void loadMovieDetailsFromNowPlaying() {
```

Comment:
rename to `processNowPlayingMoviesDetails`
integrate `processNowPlayingMoviesDetails` into the `loadNowPlayingMovies` method and pass now playing movies as parameter